angr/analyses/decompiler/optimization_passes/flip_boolean_cmp.py

```diff
@@ -8,6 +8,7 @@
 from angr.analyses.decompiler.structuring.structurer_nodes import ConditionNode
 from angr.analyses.decompiler.utils import (
     structured_node_is_simple_return,
+    structured_node_is_simple_return_strict,
```

Comment:
The naming here is a bit confusing:
* `structured_node_is_simple_return` returns `True` for sequences that may include calls, stores, etc. We should re-examine this function's uses and make sure they intend for this behavior, and pick a more descriptive function name.
* `structured_node_is_simple_return_strict` is so named because a different name would probably make you have to guess whether you should use this or the `structured_node_is_simple_return`. But I admit this choice might also leave readers confused.

Anyway, I'll defer renaming to a follow-up PR unless someone has a better idea.